app.js

```diff
@@ -25,27 +25,33 @@ app.use(bodyParser.json());
 app.use(helmet());
 app.use(limiter);
 
-mongoose.connect('mongodb://localhost:27017/mestodb', {
+mongoose.connect(SERVER_CONNECT, {
   useNewUrlParser: true,
   useCreateIndex: true,
   useFindAndModify: false,
 });
 
 app.use(requestLogger);
 
+app.get('/crash-test', () => {
+  setTimeout(() => {
+    throw new Error('Сервер сейчас упадёт');
+  }, 0);
+});
+
 app.post('/signup', celebrate({
   body: Joi.object().keys({
     name: Joi.string().required().min(2).max(30),
     about: Joi.string().required().min(2).max(30),
     avatar: Joi.string().required().regex(/^http[s]?:\/\/(www\.)?(?!(www\.))((\d{1,3}\.){3}\d{1,3}(:\d{2,5})?|([a-z-]+(\.|:\d{2,5}))+)(\/?)(([a-zA-Z0-9-]{1,}?\/?)*#?)?$/i),
     email: Joi.string().required().email(),
-    password: Joi.string().min(8).regex(/^[a-zA-Z0-9]$/i),
+    password: Joi.string().min(8).regex(/[a-zA-Z0-9]/i),
   }),
 }), createUser);
 app.post('/signin', celebrate({
   body: Joi.object().keys({
     email: Joi.string().required().email(),
-    password: Joi.string().min(8).regex(/^[a-zA-Z0-9]$/i),
+    password: Joi.string().min(8).regex(/[a-zA-Z0-9]/i),
```

Comment:
Можно лучше: убрать выражение, проверяющее пароль.
```suggestion
    password: Joi.string().min(8),
```